concrete/src/Support/CodingStyle.php

```diff
@@ -0,0 +1,557 @@
+            'include' => true,
+            // Code MUST use configured indentation type.
+            'indentation_type' => ($flags && static::FLAG_PHPONLY) === static::FLAG_PHPONLY ? true : false,
+            // Replaces is_null(parameter) expression with `null === parameter`.
+            'is_null' => [
+                'use_yoda_style' => false,
+            ],
+            // Ensure there is no code on the same line as the PHP open tag.
+            'linebreak_after_opening_tag' => ($flags && static::FLAG_PHPONLY) === static::FLAG_PHPONLY ? true : false,
+            // All PHP files must use same line ending.
+            'line_ending' => true,
+            // Cast should be written in lower case.
+            'lowercase_cast' => true,
+            // The PHP constants `true`, `false`, and `null` MUST be in lower case.
+            'lowercase_constants' => true,
+            // PHP keywords MUST be in lower case.
+            'lowercase_keywords' => true,
+            // Magic constants should be referred to using the correct casing.
+            'magic_constant_casing' => true,
+            // In method arguments and method call, there MUST NOT be a space before each comma and there MUST be one space after each comma.
+            'method_argument_space' => true,
+            // Methods must be separated with one blank line.
+            'method_separation' => true,
+            // Replaces `intval`, `floatval`, `doubleval`, `strval` and `boolval` function calls with according type casting operator.
+            'modernize_types_casting' => true,
+            // Function defined by PHP should be called using the correct casing.
+            'native_function_casing' => true,
+            // All instances created with new keyword must be followed by braces.
+            'new_with_braces' => true,
+            // Master functions shall be used instead of aliases.
+            'no_alias_functions' => true,
+            // There should be no empty lines after class opening brace.
+            'no_blank_lines_after_class_opening' => true,
+            // There should not be blank lines between docblock and the documented element.
+            'no_blank_lines_after_phpdoc' => true,
+            // The closing PHP tag MUST be omitted from files containing only PHP.
+            'no_closing_tag' => true,
+            // There should not be any empty comments.
+            'no_empty_comment' => true,
+            // There should not be empty PHPDoc blocks.
+            'no_empty_phpdoc' => true,
+            // Remove useless semicolon statements.
+            'no_empty_statement' => true,
+            // Removes extra blank lines and/or blank lines following configuration.
+            'no_extra_consecutive_blank_lines' => [
+                'tokens' => ['curly_brace_block', 'extra', 'parenthesis_brace_block', 'square_brace_block', 'throw', 'use'],
```

Comment:
This doesn't make sense to me. Can we just conform to PSR-2?

Reply:
Are you referring only to `no_extra_consecutive_blank_lines`?